Policies/Kubernetes/flexvolume-drivers/azurepolicy.json

```diff
@@ -0,0 +1,59 @@
+{
+  "properties": {
+    "displayName": "Ensure only allowed FlexVolume Drivers are used in Kubernetes Cluster",
+    "policyType": "Custom",
+    "mode": "Microsoft.Kubernetes.Data",
+    "description": "This policy ensures only allowe FlexVolume Drivers are used inside a Kubernetes cluster. For instructions on using this policy, please visit https://aka.ms/kubepolicydoc",
+    "parameters": {
+      "effect": {
+        "type": "String",
+        "metadata": {
+          "displayName": "Effect",
+          "description": "Enable or disable the execution of the policy"
+        },
+        "allowedValues": [
+          "enforceOPAConstraint",
+          "disabled"
+        ],
+        "defaultValue": "enforceOPAConstraint"
+      },
+      "excludedNamespaces": {
+        "type": "Array",
+        "metadata": {
+          "displayName": "Namespace exclusions",
+          "description": "List of Kubernetes namespaces to exclude from policy evaluation. Providing a value for this parameter is optional."
+        },
+        "defaultValue": []
+      },
+      "allowedFlexVolumes": {
```

Comment:
Can you please rename the parameter name to allowedFlexVolumeDrivers in all files to reflect the actual drivers that we are testing?

In case of gatekeeper policy, allowedFlexVolumes parameter name make sense as the parameter type is an object and there is a driver property within the object. As we are exposing directly the drivers array, it will be good to update the parameter name to reflect what is being checked.